web/public/index.html

```diff
@@ -45,7 +45,7 @@
           env: getTemplateValue('{{ .Env }}', ''),
           demoEnabled: getTemplateValue('{{ .DemoEnabled }}', '["pokeshop", "otel"]'),
           demoEndpoints: getTemplateValue('{{ .DemoEndpoints }}', '{"PokeshopHttp": "http://demo-pokemon-api.demo.svc.cluster.local", "PokeshopGrpc": "demo-pokemon-api.demo.svc.cluster.local:8082", "OtelFrontend": "http://otel-frontend:8084" }'),
-          experimentalFeatures: getTemplateValue('{{ .ExperimentalFeatures }}', '[]'),
+          experimentalFeatures: getTemplateValue('{{ .ExperimentalFeatures }}', '["transactions1"]'),
```

Comment:
is that `1` at the end of the default value (`transactions1`) ok?
Also, I would expect the default to be "no experiments enabled"

Reply:
This was a mistake @schoren Fixing it right now.